.github/workflows/buildJDK.yml

```
@@ -213,6 +225,22 @@ jobs:
           Write-Host $DIFF
           exit 1
         }
+        Set-Content -Path 'VersionTest.java' -Value "
+        import org.graalvm.home.Version;
+        public class VersionTest {
+            private static final String VERSION = Version.getCurrent().toString();
+            public static void main(String[] args) {
+                System.out.println(VERSION);
+            }
+        }
+        "
+        & $MANDREL_HOME\bin\javac -cp $MANDREL_HOME\lib\jvmci\graal-sdk.jar VersionTest.java
+        & $MANDREL_HOME\bin\native-image.cmd --initialize-at-build-time=. VersionTest
+        $VERSION=(& ./versiontest)
+        Write-Host $VERSION
+        if ("$VERSION" -NotMatch "$Env:MANDREL_VERSION") {
+          exit 1
+        }
```

Is there a reason you didn't go with?

```suggestion
        if ("$VERSION" -NotMatch "$Env:MANDREL_VERSION") {
          Write-Host got $VERSION but expected $Env:MANDREL_VERSION
          exit 1
        }
```

---

No reason. Thanks for it!